runtime/libpgmath/lib/common/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
 if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-  if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86_64|ppc64le" AND NOT ${LIBPGMATH_WITH_GENERIC})
+  if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86_64|ppc64le|aarch64" AND NOT ${LIBPGMATH_WITH_GENERIC})
```

Comment:
ok, so common is a place for things that are shared among known architectures and not built for generic...

Reply:
powi, sincos - shouldn't they move to generic at some point?
sincosf - shouldn't it move to x86_64 at some point?

Reply:
powi and sincos probably at some point when we cleanup the rest of the common directory (e.g. we move headers under `include/`, etc.). Sincosf yes it should be moved.

Reply:
powi and sincos are generic but they also get used for x86_64, power and arm. I need to think about it, because if we move them under generic then for example for x86_64 I have to add the generic dir when I build and add the logic to only build `pow` and `sincos` and ignore the rest of generic (since everything is actually under `x86_64`. I'll leave as they are for now and think about it, in a future commit will fix that plus create the `include` folder. I moved `sincosf` under `x86_64` though.